src/firebolt/async_db/cursor.py

```diff
@@ -327,25 +338,31 @@ async def _api_request(
             content=query,
         )
 
+    async def _async_api_request(self, query: str) -> Response:
```

Comment:
nit: `_async_execution_api_request`, not to confuse with asyncio

Reply:
Done.